scripts/debian/postinst.in

```diff
@@ -78,12 +78,12 @@ systemctl --system daemon-reload || true
 # If needed, try to load/compile the driver through falco-driver-loader
 case "$chosen_driver" in
     "kmod")
-      echo "[POST-INSTALL] Call 'falco-driver-loader module':"
-      falco-driver-loader module
+      echo "[POST-INSTALL] Call 'falco-driver-loader --compile module':"
+      falco-driver-loader --compile module
       ;;
     "bpf")
-      echo "[POST-INSTALL] Call 'falco-driver-loader bpf':"
-      falco-driver-loader bpf
+      echo "[POST-INSTALL] Call 'falco-driver-loader --compile bpf':"
+      falco-driver-loader --compile bpf
```

Comment:
@leogr do we need to compile the bpf probe too?

Reply:
Or even better, we might want to add the `falco-driver-loader --compile X` in falco-kmod and falco-bpf.service, as `PreStart`?

Reply:
Since in a local system, one is supposed to have all required dependencies installed (i.e., kernel headers), I'd not use the download approach. It could create more issues than benefits (for example, think in an air-gapped env).

For the kmod use case, assuming that dkms will automatically rebuild the module when the local kernel is upgraded,  I think the post-install approach is enough. It should work in all situations.

For the ebpf case, it likely won't. Think this the following situation:
- the current kernel version is X
- the user installs Falco, and the ebpf gets built for X
- the user upgrades the kernel to Y
- the system is rebooted
- Falco won't work anymore because the probe was built for X

I don't have a definitive answer yet, but the best option is likely to leave the kmod at the post-install stage and only move the bpf at the `PreStart` stage. :thinking: